modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/configuration/icon/import_object_definition.jsp

```diff
@@ -9,35 +9,45 @@
 
 <div>
 	<react:component
-		module="js/components/ModalImportObjectDefinition"
+		module="js/components/ModalImport"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
-				"importObjectDefinitionURL",
+				"apiURL", "/o/object-admin/v1.0/object-definitions/by-external-reference-code/"
+			).put(
+				"importURL",
 				PortletURLBuilder.createActionURL(
 					renderResponse
 				).setActionName(
 					"/object_definitions/import_object_definition"
 				).setRedirect(
 					currentURL
 				).buildString()
+			).put(
+				"JSONInputId", "objectDefinitionJSON"
```

Comment:
What is the meaning of this parameter?

Reply:
Hi @marcelabc, this parameter is an abstraction of this [old parameter](https://github.com/liferay/liferay-portal/blob/d8999c1b3ddfd6df38f28c6f478b63fb2a2e2983/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalImportObjectDefinition.tsx#L44) with a more generic name.

I removed the `portletNamespace` and concatenated it [here](https://github.com/liferay-objects/liferay-portal/pull/2913/commits/df6829422cd9e57dc462123e6db4de85b2a47abb#diff-28a9d78817ad22dee2bc755b7020c5b05b67d831b0103175f20a6dd7ed75f033R183)